Scripts/assignment/emme_assignment.py

```diff
@@ -763,6 +754,26 @@ def _assign_congested_transit(self, scen_id):
             transit_assignment_spec=self.transit_spec,
             congestion_function=param.trass_func,
             stopping_criteria=param.trass_stop,
-            log_worksheets=False, scenario=scenario)
+            log_worksheets=False, scenario=scenario,
+            save_strategies=True)
         self.emme_project.matrix_results(self.transit_result_spec, scenario)
         self.emme_project.logger.info("Transit assignment performed for scenario {}".format(str(scen_id)))
+
+    def _adjust_path_results(self, mtx_path, mtx_time):
+        """
+        Path analysis returns zeros values if path is not found.
+        This sets zeros to value 999999 if time matrix has infinte value, 
+        excluding internal trips.
+        """
+        self.emme_project.matrix_calc({
+            "type": "MATRIX_CALCULATION",
+            "result": mtx_path,
+            "expression": "999999*(p!=q)",
+            "constraint": {
+                "by_value": {
+                    "od_values": mtx_time,
+                    "interval_min": 0,
+                    "interval_max": 999999,
+                    "condition": "EXCLUDE"}
+                    }
+                })
```

Comment:
I have tried to avoid doing matrix operations inside emme, for efficiency and readability reasons. I think this could be done according to the same logic as for other matrix modifications done in `get_impedance()`.

Reply:
Of course! I just wonder why I didn't think of this at first place! :) The operation is much more simple and compact with Numpy.